From 75b845ff1e5fe991bad3ff39bdd639bff623e759 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 31 Jan 2009 21:04:54 +0200
Subject: [PATCH] Makefile: Cygwin compile

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,11 @@ RPM_VERLIST = $(filter-out g% dirty,$(su
 RPM_VERSION = $(word 1,$(RPM_VERLIST))
 RPM_RELEASE = $(word 2,$(RPM_VERLIST))$(if $(WTDIRTY),.dirty)
 
-LDLIBS ?= -lcurses
+ifeq (yes,$(shell uname -a | grep -i cygwin && echo yes))
+CYGWIN_LIBS = -liconv
+endif
+
+LDLIBS ?= -lcurses $(CYGWIN_LIBS)
 CFLAGS ?= -Wall -O2
 DFLAGS	= -g -DDEBUG -Werror -O0
 PROGS	= tig
