From 918025c44727a050782fb91df92925f0cb08dd60 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 31 Jan 2009 21:12:42 +0200
Subject: [PATCH] tigrc.5.txt: asciidoc fix

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 doc/tigrc.5.asciidoc |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/doc/tigrc.5.asciidoc
+++ b/doc/tigrc.5.asciidoc
@@ -8,7 +8,7 @@ tigrc - Tig configuration file
 
 SYNOPSIS
 --------
-[verse]
+
 _______________________________________________________________________
 *set*   'variable' *=* 'value'
 *bind*  'keymap' 'key' 'action'
@@ -76,7 +76,7 @@ Set command
 A few selective variables can be configured via the set command. The syntax
 is:
 
-[verse]
+
 *set* variables *=* value
 
 Examples:
@@ -304,7 +304,7 @@ Bind command
 Using bind commands keys can be mapped to an action when pressed in a given
 key map. The syntax is:
 
-[verse]
+
 *bind* 'keymap' 'key' 'action'
 
 Examples:
@@ -546,7 +546,7 @@ terminal supports color, these commands
 background combinations to certain areas. Optionally, an attribute can be
 given as the last parameter. The syntax is:
 
-[verse]
+
 *color* 'area' 'fgcolor' 'bgcolor' '[attributes]'
 
 Examples:
