From 75b845ff1e5fe991bad3ff39bdd639bff623e759 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 31 Jan 2009 21:04:54 +0200
Subject: [PATCH] Makefile: Cygwin compile

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile |   10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ kernel_name := $(shell sh -c 'uname -s 2
 prefix ?= $(HOME)
 bindir ?= $(prefix)/bin
 datarootdir ?= $(prefix)/share
-sysconfdir ?= $(prefix)/etc
+sysconfdir ?= /etc
 docdir ?= $(datarootdir)/doc
 mandir ?= $(datarootdir)/man
 # DESTDIR=
@@ -37,7 +37,11 @@ RPM_VERLIST = $(filter-out g% dirty,$(su
 RPM_VERSION = $(word 1,$(RPM_VERLIST))
 RPM_RELEASE = $(word 2,$(RPM_VERLIST))$(if $(WTDIRTY),.dirty)
 
-LDLIBS ?= -lcurses
+ifeq (yes,$(shell uname -a | grep -i cygwin && echo yes))
+CYGWIN_LIBS = -liconv
+endif
+
+LDLIBS ?= -lcurses $(CYGWIN_LIBS)
 CFLAGS ?= -Wall -O2
 DFLAGS	= -g -DDEBUG -Werror -O0
 EXE	= src/tig
@@ -59,7 +63,7 @@ override CPPFLAGS += '-DTIG_VERSION="$(V
 override CPPFLAGS += '-DSYSCONFDIR="$(sysconfdir)"'
 
 ASCIIDOC ?= asciidoc
-ASCIIDOC_FLAGS = -aversion=$(VERSION) -asysconfdir=$(sysconfdir) -f doc/asciidoc.conf
+ASCIIDOC_FLAGS = -aversion=$(VERSION) -asysconfdir=$(sysconfdir) -f doc/asciidoc.conf --unsafe
 XMLTO ?= xmlto
 DOCBOOK2PDF ?= docbook2pdf
 
