From 75b845ff1e5fe991bad3ff39bdd639bff623e759 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sat, 31 Jan 2009 21:04:54 +0200
Subject: [PATCH] Makefile: Cygwin compile

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 Makefile |   11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -13,9 +13,14 @@ kernel_name := $(shell sh -c 'uname -s 2
 # Include setting from the configure script
 -include config.make
 
+ifeq (yes,$(shell uname -a | grep -i cygwin && echo yes))
+CYGWIN_LIBS = -liconv
+endif
+
 # Optional defaults.
 # TIG_ variables are set by contrib/config.make-$(kernel_name).
-TIG_NCURSES ?= -lcurses
+TIG_NCURSES ?= -lcurses $(CYGWIN_LIBS)
+
 LDFLAGS ?= $(TIG_LDFLAGS)
 CPPFLAGS ?= $(TIG_CPPFLAGS)
 LDLIBS ?= $(TIG_NCURSES) $(TIG_LDLIBS)
@@ -24,7 +29,7 @@ CFLAGS ?= -Wall -O2 $(TIG_CFLAGS)
 prefix ?= $(HOME)
 bindir ?= $(prefix)/bin
 datarootdir ?= $(prefix)/share
-sysconfdir ?= $(prefix)/etc
+sysconfdir ?= /etc
 docdir ?= $(datarootdir)/doc
 mandir ?= $(datarootdir)/man
 # DESTDIR=
@@ -68,7 +73,7 @@ override CPPFLAGS += '-DTIG_USER_CONFIG=
 endif
 
 ASCIIDOC ?= asciidoc
-ASCIIDOC_FLAGS = -aversion=$(VERSION) -asysconfdir=$(sysconfdir) -f doc/asciidoc.conf
+ASCIIDOC_FLAGS = -aversion=$(VERSION) -asysconfdir=$(sysconfdir) -f doc/asciidoc.conf --unsafe
 XMLTO ?= xmlto
 DOCBOOK2PDF ?= docbook2pdf
 
