From: Jari Aalto <jari.aalto@cante.net>
Subject: Do not try to use network connection to validate xml

---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -249,10 +249,10 @@ doc/tigmanual.7: doc/manual.asciidoc
 	$(ASCIIDOC) $(ASCIIDOC_FLAGS) -b docbook -d article $<
 
 % : %.xml
-	$(XMLTO) man -o doc $<
+	$(XMLTO) --skip-validation man -o doc $<
 
 %.html-chunked : %.xml
-	$(XMLTO) html -o $@ $<
+	$(XMLTO) --skip-validation html -o $@ $<
 
 %.pdf : %.xml
 	$(DOCBOOK2PDF) -o doc $<
