From: Jari Aalto <jari.aalto@cante.net>
Subject: Do not try to use network connection to validate xml

---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -397,10 +397,10 @@ test/API.adoc: test/tools/libtest.sh
 	$(QUIET_ASCIIDOC)$(ASCIIDOC) $(ASCIIDOC_FLAGS) -b docbook -d article $<
 
 % : %.xml
-	$(QUIET_XMLTO)$(XMLTO) man -o doc $<
+	$(QUIET_XMLTO)$(XMLTO) --skip-validation man -o doc $<
 
 %.html-chunked : %.xml
-	$(QUIET_XMLTO)$(XMLTO) html -o $@ $<
+	$(QUIET_XMLTO)$(XMLTO) --skip-validation html -o $@ $<
 
 %.pdf : %.xml
 	$(QUIET_DB2PDF)$(DOCBOOK2PDF) -o doc $<
