From: Jari Aalto <jari.aalto@cante.net>
Subject: Do not try to use network connection to validate xml

---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -223,10 +223,10 @@ tigmanual.7: manual.txt
 	$(ASCIIDOC) $(ASCIIDOC_FLAGS) -b docbook -d article $<
 
 % : %.xml
-	$(XMLTO) man $<
+	$(XMLTO) --skip-validation man $<
 
 %.html-chunked : %.xml
-	$(XMLTO) html -o $@ $<
+	$(XMLTO) --skip-validation html -o $@ $<
 
 %.pdf : %.xml
 	$(DOCBOOK2PDF) $<
